File: Iniciante/URI 1037.pas
var
    valor : double;

begin
    read(valor);
    
    if ((valor >= 0) and (valor <= 25)) then
        writeln('Intervalo [0,25]')
    else if ((valor > 25) and (valor <= 50)) then
        writeln('Intervalo (25,50]')
    else if ((valor > 50) and (valor <= 75)) then
        writeln('Intervalo (50,75]')
    else if ((valor > 75) and (valor <= 100)) then
        writeln('Intervalo (75,100]')
    else
        writeln('Fora de intervalo')
end.

File: Iniciante/URI 1037.py
valor = None

def read_numeric():
  try:
    # read for Python 2.x
    return float(raw_input())
  except NameError:
    # read for Python 3.x
    return float(input())

valor = read_numeric()
if valor >= 0 and valor <= 25:
  print("Intervalo [0,25]")
elif valor > 25 and valor <= 50:
  print("Intervalo (25,50]")
elif valor > 50 and valor <= 75:
  print("Intervalo (50,75]")
elif valor > 75 and valor <= 100:
  print("Intervalo (75,100]")
else:
  print("Fora de intervalo")